node/task.ts

```diff
@@ -407,7 +407,7 @@ export interface EndpointAuthorization {
 
 /**
  * Gets the authorization details for a service endpoint
- * If the authorization was not set and is not optional, it will throw.
+ * If the authorization was not set and is not optional, it will specify task result as failed in case of exception.
```

Comment:
If I weren't already involved in this part of the code base, I wouldn't have been able to figure out what "specify task result as failed in case of exception" meant without reading the implementation. In particular, I would be confused by "specify" and "in case of exception". Suggestion:

```suggestion
 * If the authorization was not set and is not optional, it will set the task result to Failed.
```

Note that, in `setResult`'s JSDoc comment, that function is described as _setting_ (not specifying) the task result, and "Failed" is capitalized.